glossary.md

```diff
@@ -6,28 +6,97 @@
 * `Library`: The collection of all `responses`.
 * `Registry`: The collection of available `survey objects` for use in `responses`.
 * `Analysis`: The process of converting many survey responses to a Sankey diagram by an
-  `Analyst` interacting with the `Library`. Surveys are searched by title and `tags`.
+  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or `object metadata`.
```

Comment:
How would you define "object metadata"?

Reply:
`Object metadata` refers to the fields like `organization`, `funder`, `website` - all the fields that would be used to describe a particular `survey object`. Please feel free to update this language if there is a better way to describe it.

Reply:
I'd prefer not to create a new term for the fields on a survey object. What do you think of this?
```suggestion
  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or other survey object fields.
```

Reply:
This works